DZSwiftUI/DZSwiftUI/ContentView.swift

```diff
@@ -7,20 +7,121 @@
 
 import SwiftUI
 
-///
+/// Кнопки с Alert и ActionSheet
 struct ContentView: View {
+    
+    private enum Constants {
+        static let spacing = 20
+        static let defaultGreetingText = "Привет"
+        static let colorText = "Цвет текста"
+        static let changeText = "Цвет текста изменился"
+        static let showAlertText = "Показать Alert"
+        static let exampleAlertText = "Пример Alert"
+        static let changeColorText = "Изменить цвет"
+        static let colorButtonText = "Цвет кнопки"
+        static let showChangeColorText = "Меняем цвет?"
+        static let yesText = "Да"
+        static let showActionSheetText = "Показать ActionSheet"
+        static let exampleActionSheetText = "Пример ActionSheet"
+        static let changeLanguageText = "Изменить язык приветствия"
+        static let chooseLanguageText = "Выберите язык"
+        static let russiaLanguageText = "Русский"
+        static let englandLanguageText = "Английский"
+        static let armeniaLanguageText = "Армянский"
+        static let englandHelloText = "Hello"
+        static let russiaHelloText = "Привет"
+        static let armeniaHelloText = "Հեյ"
+    }
+    
+    @State private var isShowAlertOne = false
+    @State private var isShowAlertTwo = false
+    @State private var isTextColor = false
+    @State private var textColor = Color(.blue)
+    @State private var isShowActionSheetOne = false
+    @State private var isShowActionSheetTwo = false
+    @State private var greetingText = Constants.defaultGreetingText
+    @State private var changeText = Constants.colorText
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: CGFloat(Constants.spacing)) {
+            blockOne()
+            blockTwo()
+            blockThree()
+            blockFour()
+        }
+    }
+    
+    private func blockOne() -> some View {
```

Comment:
```suggestion
    private var blockOneView: some View {
```
Если нет параметров переделываем в такое вычисляемое, поправь везде так же ниже